Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Melee/fun.yml

```
@@ -0,0 +1,16 @@
+
+- type: entity
+  parent: BarberScissors
+  id: SuperBarberScissors
+  name: barber scissors
+  suffix: ADMINBUS TOOL, DO NOT MAP
+  categories: [ DoNotMap ]
+  description: Is able to reshape the hairstyle of any crew cut to your liking. Also can remove all of their hair in one strike.
+  components:
+  - type: BaldifyOnHit
+  - type: SignOnHit
+    signSprite: _Goobstation/Objects/Misc/baldsign.rsi
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)